lambdas/process_webhooks/process_webhooks.py

```diff
@@ -71,6 +71,24 @@ def _add_gh_header(data_object, headers):
     return headers
 
 
+def _is_from_queue(event):
+    """
+    Check to see if this webhook is being sent from the SQS queue.
+    This is important to avoid duplicating the hook in the queue
+    in the event of a failure.
+    """
+    from_queue = False
```

Comment:
nit: you could `return event.get('from_queue') == "True"` Perhaps that's a little too terse tho? Up to you.